### DISCUSSION TITLE
DiscordChannel Choice

### DISCUSSION THREAD

**Discussion by 1stNeferpitou** [DSharpPlus/DSharpPlus#1606] [Support] [answered]
using DiscordChannel attribute shows only top 8 channels not all

**Comment by Plerx2493** [comment]
If you talk about the slashcommand option. It only functions like a autocompletion. Start typing the name of a not shown channel

**Reply by OoLunar** [reply, marked as answer]
Marking this discussion as solved since we found a general conclusion on the discord: https://discord.com/channels/379378609942560770/1143716696213966998
